tests/Agent/UnitTests/Core.UnitTest/WireModels/ErrorTraceWireModelTests.cs

```diff
@@ -77,7 +77,8 @@ private static IConfiguration CreateMockConfiguration()
                 Mock.Arrange(() => configuration.CaptureAttributes).Returns(true);
                 Mock.Arrange(() => configuration.CaptureAttributesExcludes)
                     .Returns(new List<string>() { "identity.*", "request.headers.*", "response.headers.*" });
-                Mock.Arrange(() => configuration.CaptureRequestParameters).Returns(true);
+                Mock.Arrange(() => configuration.CaptureAttributesIncludes).Returns(new string[] { "request.parameters.*" });
```

Comment:
Is there a reason why we need to set this  ? Tests in this files seems to pass without this line of code.

Reply:
I think we removed this because the `CaptureRequestParameters` boolean no longer exists (and doesn't really make sense anymore since parameters being captured is no longer an all-or-nothing thing).

Reply:
oh ic, I misunderstood the change. The change was to replace the existing line of code with the equivalence one. The old line of code didn't seem to really do anything to the tests.